### DISCUSSION TITLE
Web browser not launching with uninstalled live USB

### DISCUSSION THREAD

**Discussion by CuriousCoconut** [elementary/triage#611] [Q&A] [answered]
The web app won't launch when I run the new dailies or even the stable builds off a live USB.

**Comment by teamcons** [comment]
can you try opening a terminal, entering 
> flatpak run org.gnome.epiphany

and paste the output here please

**Comment by CuriousCoconut** [comment]
I had to write it out because I couldn't open my email.

The output is

error: app/org.gnome.epiphany/x86_64/master not installed

**Comment by teamcons** [comment]
Huh. Weird. I gotta look into this

**Comment by CuriousCoconut** [comment]
Thank you glad I could help at least a bit

**Comment by teamcons** [comment]
Okay, yes, someone else reported the issue, and it isnt just Web - in demo mode some apps do not launch
can you try opening other apps and tell which one isnt starting ?

it is a permission issue, in the packaging format of those app. A bug because of too tight security in a sense. Im seeing if i can bring some attention on this one.

thank you !

**Comment by CuriousCoconut** [comment]
Sure ...

App Center - launches
Calculator - does not
Calendar - launches
Camera - does not
Code - launches
Document Viewer - does not
Feedback - launches
Files - launches
GParted - launches
Install Elementary OS - launches
Mail - launches
Multitasking View - launches
Music - does not
Photos - launches
Screenshot - does not
Shortcuts - launches
System Settings - launches
Tasks - launches
Terminal - launches
Videos - does not
Web - does not

**Reply by teamcons** [reply]
Amazing. Thank you ! :) Ill copy this on the open issue

**Reply by ryonakano** [reply, marked as answer]
The apps that can't be launched are all provided by Flatpak, so this issue sounds like https://github.com/elementary/triage/issues/600

**Reply by teamcons** [reply]
> The apps that can't be launched are all provided by Flatpak, so this issue sounds like #600

Yes, that is indeed the list and details i provided in the mentioned bug request.

**Comment by CuriousCoconut** [comment]
Ok, thanks for the clarification.
